.github/workflows/on_docs_update.yml

```diff
@@ -0,0 +1,22 @@
+name: "On docs update"
+on:
+  push:
+    paths:
+    - 'docs/**'
+
+jobs:
+  trigger_docker_build:
+    name: Trigger decidim/documentation build
+    runs-on: ubuntu-latest
+    if: "github.ref == 'refs/heads/chore/trigger-docs-build'"
+    timeout-minutes: 60
+    steps:
+    - name: Send dispatch for trigger_build workflow
+      run: |
+        curl --request POST \
+        --user "decidim-bot:${{ secrets.DOCKER_WORKFLOW_PAT }}" \
```

Comment:
@oliverbarnes hi! I'm trying to trigger a workflow from this repo to decidim/documentation, but I suspect I'm sending something wrong. Specially in this line, should I create a new PAT for the `decidim-bot` user? Can you give me a help with that?

Thanks!

Reply:
Sorry, I missed the notification for this message somehow :/ That's right, `decidim-bot` needs a PAT saved as a secret (`DOCS_GEN_WORKFLOW` perhaps?) on decidim/documentation. And it also needs to be have access to the repo - I've gone ahead and given it just now.

I don't have `decidim-bot`'s credentials in order to generate its PAT, though. @andreslucena does

BTW, Andrés - I think you might have created the original `DOCKER_WORKFLOW_PAT` under your own user, since `decidim-bot` doesn't have access to decidim/docker, and the workflow dispatch still works with it 🤔 😄

Reply:
> `decidim-bot` needs a PAT saved as a secret

I've created and send it privately to @mrcasals 

> on decidim/documentation. 

PATs belong to the user AFAIK 

> I've gone ahead and given it just now.

I've removed and added it to the group that belongs (developers) as it's easier to manage groups vs users (at least for me)  --- maybe with decidim-bot as it's special it'd be better to have it configured to the repostiories that it need access? 

> BTW, Andrés - I think you might have created the original DOCKER_WORKFLOW_PAT under your own user, since decidim-bot doesn't have access to decidim/docker, and the workflow dispatch still works with it thinking smile

PAT belongs to the user settings account https://github.com/settings/tokens - in the decidim-bot account we have these:

* Documentation generator action
* Crowdin Decidim.org 
* Container Registry
* Docker Workflow 
* Crowdin
---

decidim-bot belongs to developers group: https://github.com/orgs/decidim/teams/developers/members 
developers-group has read access to docker repository: https://github.com/orgs/decidim/teams/developers/repositories 
I think with read access should work in this case also, as I'm seeing that with that:
 
> Can read and clone this repository. Can also open and comment on issues and pull requests. 

Maybe it don't need read access and just with the PAT is enough, as we're always working with public repositories, but I'm not sure.

Reply:
Yes, Personal Access Token belongs to a user :) and that user needs to have access to the repo for a secret created with their PAT to work. If `decidim-bot` is now part of a group with global access to the Decidim repos, its PAT should work wherever it's added as a secret now. Agree it makes more sense to manage it in a group. Not sure that should be developers or maintainers though - I think it needs more than read access.

My asking about whether you created the PAT under your own personal account came from not seeing `decidim-bot` inside any of the decidim/docker groups or users, but still the `DOCKER_WORKFLOW_PAT` seemed to be working there. Only way I could see for that to be the case would be for it to have been created under your own user.

Reply:
> I think it needs more than read access

You're right!! I thought that with read access was enough but there was an error saying that it needs admin rights... So we're back to were we where (with decidim-bot with admin rights in the repo 😅)

> My asking about whether you created the PAT under your own personal account came from not seeing decidim-bot inside any of the decidim/docker groups or users, but still the DOCKER_WORKFLOW_PAT seemed to be working there. Only way I could see for that to be the case would be for it to have been created under your own user.

Yes, that's weird actually.... I think it could be because packages permissions aren't in the repository, only in the organization or something like that: the PAT has permissions to `write: packages` but we don't configure this rule of allowing the package <-> repository write and publish to this user/group nowhere 🤷🏽

Reply:
Ahh, that could be. The registry is indeed its own thing at the org level